src/engine/qcommon/net_ip.cpp

```diff
@@ -1948,8 +1877,12 @@ NET_Restart_f
 */
 void NET_Restart_f()
 {
-	NET_Config( false );
+	NET_DisableNetworking();
 	SV_NET_Config();
 	Net::ShutDownDNS();
-	NET_Config( true );
```

Comment:
So this was a bug?

Reply:
It's not really doing something different. The diff may be confusing because the boolean argument on the left side is `networkingEnabled` and on the right side it is `serverMode` (the responsibility of deciding serverMode was put on the caller).